src/scrape_up/github/__init__.py

```diff
@@ -1,5 +1,130 @@
+import requests
```

Comment:
@ayushanand308 
Do not make changes to `__init__.py`, there's a separate file created for `Users`, `Repository`, etc. Add this method in the repository class of the `github/repository.py` file.

Reply:
Looks like you haven't updated the forked repository, sync it with the current project status and also update in your computer.